java/helidon-mp/src/main/java/web/helidon/UserResource.java

```diff
@@ -0,0 +1,28 @@
+package web.helidon;
+
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+
+@Path("/user")
+public class UserResource {
+
+    @Path("/{id}")
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
```

Comment:
should be text plain 

If you sport any error if other implementations, do not hesitate to PR